hax/hax/handler.py

```diff
@@ -158,6 +158,29 @@ def _update_process_status(self, p: WorkPlanner, motr: Motr,
             except Exception as e:
                 LOG.warning("Send event failed due to '%s'", e)
 
+    def cache_process_event(self, state: HAState) -> None:
+
+        def _to_json() -> str:
```

Comment:
May be this can be moved to utils, `ha_state_to_json(fid: Fid, state: str) -> str`

Reply:
since it is hastate to json.. passing HA State would make more sense?